server/Dockerfile

```diff
@@ -145,59 +101,67 @@ FROM "${DOCKER_HUB_PROXY}debian:bullseye-slim"
+    RUN <<-EOF
+            if [ ! -z "${MISP_COMMIT}" ]; then
+                git clone https://github.com/MISP/MISP.git /var/www/MISP && cd /var/www/MISP && git checkout "${MISP_COMMIT}"
+            else
+                git clone --branch "${MISP_TAG}" --depth 1 https://github.com/MISP/MISP.git /var/www/MISP
+            fi
+            cd /var/www/MISP || exit; git -c submodule."PyMISP".update=none submodule update --init --recursive .
+            rm -r /var/www/MISP/PyMISP
+            find /var/www/MISP/INSTALL/* ! -name 'MYSQL.sql' -type f -exec rm {} +
+            find /var/www/MISP/INSTALL/* ! -name 'MYSQL.sql' -type l -exec rm {} +
+            rm -rf /var/www/MISP/.git
+            mkdir -p /var/www/MISP/app/Vendor && mkdir -p /var/www/MISP/app/Plugin
+EOF
 
     # Python Modules
     COPY --from=python-build /wheels /wheels
     RUN pip3 install --no-cache-dir /wheels/*.whl && rm -rf /wheels
 
     # PHP: install prebuilt libraries, then install the app's PHP deps
-    COPY --from=php-build /usr/lib/php/${PHP_VER}/ssdeep.so /usr/lib/php/${PHP_VER}/ssdeep.so
-    COPY --from=php-build /usr/lib/php/${PHP_VER}/rdkafka.so /usr/lib/php/${PHP_VER}/rdkafka.so
-    COPY --from=php-build /usr/lib/php/${PHP_VER}/brotli.so /usr/lib/php/${PHP_VER}/brotli.so
-    COPY --from=php-build /usr/lib/php/${PHP_VER}/simdjson.so /usr/lib/php/${PHP_VER}/simdjson.so
+    COPY --from=php-build ["/usr/lib/php/${PHP_VER}/ssdeep.so", "/usr/lib/php/${PHP_VER}/rdkafka.so", "/usr/lib/php/${PHP_VER}/brotli.so", "/usr/lib/php/${PHP_VER}/simdjson.so", "/usr/lib/php/${PHP_VER}/"]
 
     # Do an early chown to limit image size
     COPY --from=composer-build --chown=www-data:www-data /tmp/Vendor /var/www/MISP/app/Vendor
     COPY --from=composer-build --chown=www-data:www-data /tmp/Plugin /var/www/MISP/app/Plugin
 
-    RUN for dir in /etc/php/*; do echo "extension=ssdeep.so" > "$dir/mods-available/ssdeep.ini"; done; phpenmod ssdeep
-    RUN for dir in /etc/php/*; do echo "extension=rdkafka.so" > "$dir/mods-available/rdkafka.ini"; done; phpenmod rdkafka
-    RUN for dir in /etc/php/*; do echo "extension=brotli.so" > "$dir/mods-available/brotli.ini"; done; phpenmod brotli
-    RUN for dir in /etc/php/*; do echo "extension=simdjson.so" > "$dir/mods-available/simdjson.ini"; done; phpenmod simdjson
-    RUN phpenmod redis
+    # Gather these in one layer, only act on actual directories under /etc/php/
+    RUN <<-EOF
+            set -- "ssdeep" "rdkafka" "brotli" "simdjson"
+            for mod in "$@"; do
+                for dir in /etc/php/*/; do
+                    echo "extension=${mod}.so" > "${dir}mods-available/${mod}.ini"
+                done;
+                phpenmod "${mod}"
+            done;
+            phpenmod redis
+EOF
 
     # nginx
     RUN rm /etc/nginx/sites-enabled/*; mkdir /run/php /etc/nginx/certs
-    COPY files/etc/nginx/misp /etc/nginx/sites-available/misp
```

Comment:
Handled in the final COPY

Reply:
Where?

Reply:
It happens [here](https://github.com/ostefano/docker-misp/pull/25/files#diff-7dcfbc368ef61cdb87ff87938d3901a82097fe398633f995dc4ab1cbee761a40R179) due to shuffling of files under `files/`